<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommAudioClass</class>
 <widget class="QMainWindow" name="CommAudioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommAudio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Currently Playing: </string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1071</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelUsers">
         <property name="text">
          <string>Connected Users:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelRemoteSongs">
         <property name="text">
          <string>Available Songs:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelLocalSongs">
         <property name="text">
          <string>Your Songs:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="treeUsers">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Role</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QTreeWidget" name="treeLocalSongs">
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string>Song</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTreeWidget" name="treeRemoteSongs">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string>Song</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Owner</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelCurrentTime">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderProgress">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTotalTime">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>239</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controlButtonSlider">
     <item>
      <widget class="QPushButton" name="btnPrevSong">
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlaySong">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNextSong">
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="volumeLayout">
     <item>
      <widget class="QLabel" name="labelVolume">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderVolume">
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionHostSession"/>
    <addaction name="actionJoinSession"/>
    <addaction name="actionLeaveSession"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPublicSongFolder"/>
    <addaction name="actionDownloadFolder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSession"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionJoinSession">
   <property name="text">
    <string>Join Session</string>
   </property>
  </action>
  <action name="actionHostSession">
   <property name="text">
    <string>Host Session</string>
   </property>
  </action>
  <action name="actionPublicSongFolder">
   <property name="text">
    <string>Change Public Song Folder</string>
   </property>
  </action>
  <action name="actionDownloadFolder">
   <property name="text">
    <string>Change Download Folder</string>
   </property>
  </action>
  <action name="actionLeaveSession">
   <property name="text">
    <string>Leave Session</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CommAudio.qrc"/>
 </resources>
 <connections/>
</ui>
